diff --git a/calicoctl/commands/version.go b/calicoctl/commands/version.go
index 48401708..132e8d7d 100644
--- a/calicoctl/commands/version.go
+++ b/calicoctl/commands/version.go
@@ -22,11 +22,11 @@ import (
 	"github.com/docopt/docopt-go"
 )
 
-var VERSION, BUILD_DATE, GIT_REVISION string
+var VERSION string
 var VERSION_SUMMARY string
 
 func init() {
-	VERSION_SUMMARY = "calicoctl version " + VERSION + ", build " + GIT_REVISION
+	VERSION_SUMMARY = "calicoctl version " + VERSION
 }
 
 func Version(args []string) {
@@ -49,6 +49,4 @@ Description:
 	}
 
 	fmt.Println("Version:     ", VERSION)
-	fmt.Println("Build date:  ", BUILD_DATE)
-	fmt.Println("Git commit:  ", GIT_REVISION)
 }
